src/drivers/postgres/impls/cqrs/queries.py

```diff
@@ -0,0 +1,37 @@
+from src.drivers.postgres.impls.cqrs.common import execute_sql
+
+
+def get_all_customers():
+    sql = 'select * from customers'
+    return execute_sql(sql)
+
+
+def get_customer_by_id(customer_id):
+    sql = f"select * from customers where id=%s"
```

Comment:
I'm wondering about sql injections a little. I guess we don't have to consider it in RnD. :dunno: